website/content/commands/acl/policy/read.mdx

```diff
@@ -7,8 +7,18 @@ page_title: 'Commands: ACL Policy Read'
 
 Command: `consul acl policy read`
 
+Corresponding HTTP API Endpoint: [\[GET\] /v1/acl/policy/:id](https://www.consul.io/api-docs/acl/policies#read-a-policy)
```

Comment:
This command technically has two different corresponding HTTP API endpoints:
- Read a policy: https://www.consul.io/api-docs/acl/policies#read-a-policy
- Read a policy by name: https://www.consul.io/api-docs/acl/policies#read-a-policy-by-name

Should we list both? Or just this one?

Reply:
Elsewhere in this PR, a similar endpoint actually listed both endpoints. That seems like a reasonable approach. Let's apply here as well.

From `/commands/acl/role/read`
> Corresponding HTTP API Endpoints: [\[GET\] /v1/acl/role/:id](https://www.consul.io/api-docs/acl/roles#read-a-role), [\[GET\] /v1/acl/role/name/:name](https://www.consul.io/api-docs/acl/roles#read-a-role-by-name)

Reply:
I've made this consistent across the docs with the comma separated list.